### DISCUSSION TITLE
异常白屏

### DISCUSSION THREAD

**Discussion by tututu411** [galacean/discussion#75] [Editor]
![image](https://github.com/galacean/discussion/assets/65388278/335297a8-6599-4d02-a3b7-8d881fee2895)
白屏前操作步骤：创建材质-选择基础纹理-基础纹理源texture删除

**Comment by cptbtptpbcptdtptp** [comment]
👌这个问题我们正在跟进，可以给我们拷贝一下出错的项目吗，方便我们检查问题。
